atlasdb-config/src/main/java/com/palantir/atlasdb/config/ServerListConfigs.java

```diff
@@ -27,11 +26,13 @@ private ServerListConfigs() {
     }
 
     public static ServerListConfig parseInstallAndRuntimeConfigs(TimeLockClientConfig installClientConfig,
-            Supplier<Optional<TimeLockRuntimeConfig>> runtimeConfig,
+            Supplier<TimeLockRuntimeConfig> runtimeConfig,
             String namespace) {
-        ServerListConfig nonNamespacedConfig = runtimeConfig.get()
-                .map(TimeLockRuntimeConfig::serversList)
-                .orElse(installClientConfig.serversList());
+        ServerListConfig nonNamespacedConfig = installClientConfig.serversList();
```

Comment:
This is a behaviour change. Previously, if we have 0 servers in the runtime block and servers in the install config, we picked the runtime block.

With the change of making the runtime block mandatory, there is now no distinction between "not actually configured block" and "configured block that has no servers at the moment".

We may want to deprecate the install config servers list, so that it's no longer valid to have a non-empty list of servers in the install config.